web/source/osk/visualKeyboard.ts

```diff
@@ -936,6 +972,8 @@ namespace com.keyman.osk {
       if(this.keyPending) {
         this.highlightKey(this.keyPending,false);
 
+        let touchKbdPos = this.getTouchCoordinatesOnKeyboard(e.changedTouches[0]);
```

Comment:
Currently, this is an unused variable, scoped solely to `if (this.keyPending) { }` branch. What is the intended purpose of this assignment?

Reply:
While I was designing the content of this PR, I found it helpful to remember what the end goal of the calculations would be.  Note the `FIXME` section a little higher up the code page - that code would be placed _here_ longterm and would send the data elsewhere (in future PRs) for use with the LMLayer.